core/src/main/kotlin/apiDocs/RunsDocs.kt

```diff
@@ -193,3 +196,86 @@ val getVulnerabilitiesByRunId: OpenApiRoute.() -> Unit = {
+                                    Severity.ERROR,
+                                    "License GPL-1.0-or-later found for package 'Maven:org.glassfish.jersey." +
+                                        "media:jersey-media-jaxb:2.42' is categorized as strict-copyleft which " +
+                                        "must not be used for BT05 Client application applications.",
+                                    "Find more information about the [strict-copyleft](https://inside-" +
+                                        "docupedia.bosch.com/confluence/x/2gNahg) and the [strict-copyleft-with-saas]" +
+                                        "(https://inside-docupedia.bosch.com/confluence/x/4ANahg) categories.\n" +
+                                        "Information about this policy and how to fix this issue can be found " +
+                                        "[here](https://inside-docupedia.bosch.com/confluence/x/pdOXc).\n\n" +
+                                        "If this is a false-positive or ineffective finding, it can be fixed in your " +
+                                        "`.ort.yml` file:\n\n" +
+                                        "```yaml\n" +
+                                        "---\n" +
+                                        "package_configurations:\n" +
+                                        "- id: \"Maven:org.glassfish.jersey.media:jersey-media-jaxb:2.42\"\n" +
+                                        "  source_artifact_url: \"https://repo.maven.apache.org/maven2/org/glassfish/" +
+                                        "jersey/media/jersey-media-jaxb/2.42/jersey-media-jaxb-2.42-sources.jar\"\n" +
+                                        "  license_finding_curations:\n" +
+                                        "  - path: \"META-INF/NOTICE.md\"\n" +
+                                        "    start_lines: \"84\"\n" +
+                                        "    line_count: 1\n" +
+                                        "    detected_license: \"GPL-1.0-or-later\"\n" +
+                                        "    concluded_license: <Insert correct license.>\n" +
+                                        "    reason: <Choose one of the reason from the list [CODE, DATA_OF, " +
+                                        "DOCUMENTATION_OF, INCORRECT, NOT_DETECTED, REFERENCE].>\n" +
+                                        "    comment: \"<Describe the reason for the license finding curation.>\"\n" +
+                                        "\n" +
+                                        "```\n" +
+                                        "Documentation how to configure package configurations in the `.ort.yml` " +
+                                        "file can be found\n" +
+                                        "[here](https://oss-review-toolkit.org/ort/docs/configuration/ort-yml" +
+                                        "#curating-project-license-findings)."
```

Comment:
Do not use Bosch internals like "OCaaS Policy C1 Strict Copyleft" or "BT05 Client application".

For the example I would actually use a shorter description, like the rules [here](https://github.com/oss-review-toolkit/ort-config/blob/61c1d289bdd29504286cc87b7d9935b5d9824834/evaluator.rules.kts#L1150) would create.

Reply:
I've put some generic messages that is not bosch-related.